src/pages/molecules/footer/Footer.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
```

Comment:
If there are common molecules/ organims which will be used across many pages please keep it outside pages folder.
Inside pages folder, there should be only pages folders nothing else, inside each pages we can have their respective organisms/ molecules
SImilarly for all levels